database/buildSchema/43_views_functions_triggers.sql

```diff
@@ -1397,6 +1397,8 @@ STABLE;
 
 -- REVIEWER_ACTION_LIST
 -- Aggregated VIEW method of reviewer action to each application on application list page
+DROP FUNCTION IF EXISTS review_list (integer, integer, public.application_status);
```

Comment:
Somehow we've ended up with two different "review_list" functions in the database.

Can you drop the other one too:
```
DROP FUNCTION IF EXISTS review_list (integer, integer);
```

Reply:
Oh! I see. Thanks!